From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@mthalman-msft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mthalman). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

mthalman-msft commented:
@tysonn: requesting merge

tysonn commented:
@mthalman-msft - This PR has a merge conflict. Probably the easiest way out is to close and resubmit.

carolinacmoravia committed:
C8475: Hardcoded EN locale in link

Hello, @UnifyCloud,

Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

WilliamAntonRohm committed:
Merge pull request #75 from MicrosoftDocs/master

bringing thhomee sheep

WilliamAntonRohm committed:
Merge pull request #76 from MicrosoftDocs/master

bring

WilliamAntonRohm committed:
adding section for linked server connections

WilliamAntonRohm committed:
minor editing for AcroLinx

WilliamAntonRohm committed:
adding SSMS download link

mattbriggs committed:
adding HLH note

mattbriggs committed:
adding bracket

elisolMS committed:
clarified guidance and fixed link

adegeo committed:
Added new article for logging

adegeo committed:
Adjusted TOC title

adegeo committed:
minor fixes

billmath committed:
updating

billmath committed:
updating

mattbriggs committed:
replace with new content

billmath committed:
updating

cherylmc committed:
new

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into create2

billmath committed:
fixing urls

v-nagta committed:
USER STORY 1220533 SaaS App Tutorial: OpsGenie Tutorial Update

v-nagta committed:
update

billmath committed:
updating

v-geberr committed:
toc and landing page for regex and hier entity quickstarts

cherylmc committed:
format

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into create2

billmath committed:
updating

billmath committed:
adding redirectors and renames

billmath committed:
fixing urls

billmath committed:
updating

adegeo committed:
added azure

billmath committed:
updating URLs

j-martens committed:
AML: redirect what is ML to what is ML Studio

billmath committed:
fixing redirects

j-martens committed:
AML: SEO improvements between AML Services and AML Studio at Carolyn's behest

KumudD committed:
removed preview text

j-martens committed:
AML: clean up topic description of what is AML Services

bsiva committed:
removed support for ReiserFS

adegeo committed:
added azure

PRMerger16 committed:
Merge pull request #36485 from KumudD/removepreviewtext2

removed preview text

davidmu1 committed:
Update active-directory-b2c-faqs.md

Corrected product name.

j-martens committed:
AML: fix link warning

mattbriggs committed:
adding semi colon

hpsin committed:
Clarify when client_secret is appropriate

Addresses #6404

hpsin committed:
Update active-directory-protocols-oauth-code.md

GitHubber17 committed:
Merge pull request #36306 from Thraka/sf-health

Added new article for logging

j-martens committed:
AML: clean up header

cherylmc committed:
update

davidmu1 committed:
Added clarification

billmath committed:
updating

mattbriggs committed:
Merge branch 'master' into mdb-pep-update

cherylmc committed:
update

Commit:
Merge pull request #36489 from davidmu1/patch-1

Update active-directory-b2c-faqs.md

Commit:
Merge pull request #36491 from hpsin/patch-2

Clarify when client_secret is appropriate

KumudD committed:
fixed links

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into pr6402

charwen committed:
Removed conflicting info.

billmath committed:
updating

PRMerger13 committed:
Merge pull request #36494 from KumudD/pr6402

Pr6402

GitHubber17 committed:
Merge pull request #36445 from cherylmc/create2

Create2

billmath committed:
fixing urls

amitbapat committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

j-martens committed:
AML: TOC clean-up

elisolMS committed:
Merge remote-tracking branch 'upstream/master' into update-tenant-article

PRMerger17 committed:
Merge pull request #36495 from charwen/patch-50

Removed conflicting info.

amitbapat committed:
Added explanation for billing implications

Commit:
rewording for clarity

GitHubber17 committed:
Merge pull request #36486 from j-martens/master

 AML: SEO improvements between AML Services and AML Studio at Carolyn'…  …

mattbriggs committed:
try to fix number issue

WilliamAntonRohm committed:
addressing review comments

charwen committed:
Added a reference to Office's HA/DR recommendation

v-shils committed:
Merge pull request #36473 from v-geberr/0328-toc-landingpage-updates

toc and landing page for regex and hier entity quickstarts

mattbriggs committed:
adding update

Commit:
Merge pull request #36498 from Blackmist/freshness

rewording for clarity

bsiva committed:
Updating Ubuntu supported versions

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into hashsync

billmath committed:
updating redirectors

bsiva committed:
resolving merge conflicts

GitHubber17 committed:
Merge pull request #35996 from WilliamAntonRohm/W-cosmos-linked-server

querying Azure Cosmos DB from SQL Server Management Studio -- create a linked server

PRMerger13 committed:
Merge pull request #30067 from carolinacmoravia/patch-511

C8475: Hardcoded EN locale in link

bsiva committed:
resolving merge conflicts

HJToland3 committed:
pre-req updates

mmacy committed:
[ACI] promote DNS name label

HJToland3 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into localworking

HJToland3 committed:
pre-req update 2

elisolMS committed:
updating broken links

Commit:
updated for missing redirect

hpsin committed:
Remove groups claim.  

Groups isn't in public preview yet and may not fully work as expected.

Ja-Dunn committed:
Merge pull request #36468 from v-nagta/opsgenie

USER STORY 1220533 SaaS App Tutorial: OpsGenie Tutorial Update

j-martens committed:
AML: remove reference to ds for beginners

PRMerger13 committed:
Merge pull request #36501 from bsiva/master

updating Ubuntu and Debian supported kernels matrices

mattbriggs committed:
clear up broken note

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-updates

rjagiewich committed:
Merge pull request #36417 from billmath/hashsync

updating

PRMerger17 committed:
Merge pull request #36506 from HJToland3/master

DMS Prerequisites update

Ja-Dunn committed:
Merge pull request #36507 from jeffgilb/328

updated for missing redirect

PRMerger18 committed:
Merge pull request #36508 from hpsin/patch-3

Remove groups claim.

mmacy committed:
[ACI] remove marketing-speak

douglaslMS committed:
Removed section about deprecated Alerts from two articles.

kgremban committed:
Add note about ARM processors for ML module

Commit:
Merge pull request #36510 from j-martens/master

AML: remove reference to ds for beginners

charwen committed:
Added HA FAQ for Office365

Commit:
Merge pull request #36499 from charwen/patch-51

Added a reference to Office's HA/DR recommendation

nitinme committed:
minor update

Commit:
Merge pull request #36516 from charwen/patch-53

Added HA FAQ for Office365

PRMerger12 committed:
Merge pull request #36496 from amitbapat/akv-soft-delete-update

Akv soft delete update

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

rjagiewich committed:
Merge pull request #36515 from douglaslMS/adf-rem-v1alerts

Removed section about deprecated Alerts from two articles.

Ja-Dunn committed:
Merge pull request #36497 from elisolMS/update-tenant-article

clarified guidance and fixed link

MikeRayMSFT committed:
Clarifying backend pool

Ja-Dunn committed:
Merge pull request #36505 from mmacy/aci-dns-promote

[ACI] promote DNS name label

PRMerger14 committed:
Merge pull request #36521 from kgremban/mar27-5717

Add note about ARM processors for ML module

v-alje committed:
Merge pull request #36522 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger20 committed:
Merge pull request #36518 from nitinme/nitinme-hdi-updates

Adding clarity for a step

PRMerger20 committed:
Merge pull request #36523 from MikeRayMSFT/20180328_AddInstructionsUnderAG

Clarifying backend pool - Fixes MicrosoftDocs/azure-docs#5511

GitHubber17 committed:
Merge pull request #36051 from mattbriggs/mdb-pep-update

 Constrained Language Mode on HLH
